### DISCUSSION TITLE
Install debian dependencies as is with requirements.txt

### DISCUSSION THREAD

**Discussion by mateodurante** [certtools/intelmq#2234] [Ideas]
Hi,

I have a problem when working with dockerized intelmq instances, the bots have system dependencies, not just pip packages.

So I propose to solve the problem in the same way that the REQUIREMENTS.txt is solved for pip dependencies defining a similar file, such as REQUIREMENTS-DEBIAN.txt, inside the directory of each bot with the list of packages to install via apt install.

**Comment by sebix** [comment]
The IntelMQ Docker container should already contain all necessary and optional dependencies, both system packages and python packages: https://github.com/certat/intelmq-docker/blob/main/.docker/intelmq-full/Dockerfile

Which package exactly is missing?

**Reply by mateodurante** [reply]
The ssh package to use rsync collector is missing.

I found this change useful, so each bot manages the packages it requires. I find it more dynamic when importing third party bots without the need to rebuild the intelmq image or manually install packages on the server.

**Reply by sebix** [reply]
Thanks!

I have create a PR to add ssh: https://github.com/certat/intelmq-docker/pull/13

**Reply by sebix** [reply]
@waldbauer-certat just merged the PR and ssh will be part of the next uploaded image.
